src/debug_utils.cc

```diff
@@ -437,6 +441,44 @@ std::vector<std::string> NativeSymbolDebuggingContext::GetLoadedLibraries() {
   return list;
 }
 
+void FWrite(FILE* file, const std::string& str) {
+  auto simple_fwrite = [&]() {
+    fwrite(str.data(), str.size(), 1, file);
```

Comment:
Maybe add a comment here explaining that the return value is deliberately ignored (at least, I assume it's deliberate.)

Reply:
Sure, done